Example does not compile

While trying to run the example:
```rs
use deno_ast::parse_module;
use deno_ast::MediaType;
use deno_ast::ParseParams;
use deno_ast::SourceTextInfo;
use std::sync::Arc;

let source_text = Arc::new("class MyClass {}");
let text_info = SourceTextInfo::new(source_text);
let parsed_source = parse_module(ParseParams {
  specifier: "file:///my_file.ts".to_string(),
  media_type: MediaType::TypeScript,
  text_info,
  capture_tokens: true,
  maybe_syntax: None,
  scope_analysis: false,
}).expect("should parse");

// returns the comments
parsed_source.comments();
// returns the tokens if captured
parsed_source.tokens();
// returns the module (AST)
parsed_source.module();
// returns the `SourceTextInfo`
parsed_source.source();
```
I get an error saying:
```
error[E0308]: mismatched types
  --> src/parsers/javascript/parser.rs:32:45
   |
32 |         let text_info = SourceTextInfo::new(source_text);
   |                                             ^^^^^^^^^^^ expected `str`, found `&str`
   |
```

I've tried casting the string, but I can only ever get it to be a `&str`. Any help is appreciated, I'm not sure what I'm going wrong.

--- a/README.md
+++ b/README.md
@@ -11,10 +11,9 @@ use deno_ast::parse_module;
 use deno_ast::MediaType;
 use deno_ast::ParseParams;
 use deno_ast::SourceTextInfo;
-use std::sync::Arc;
 
-let source_text = Arc::new("class MyClass {}");
-let text_info = SourceTextInfo::new(source_text);
+let source_text = "class MyClass {}";
+let text_info = SourceTextInfo::new(source_text.into());
 let parsed_source = parse_module(ParseParams {
   specifier: "file:///my_file.ts".to_string(),
   media_type: MediaType::TypeScript,
@@ -31,5 +30,5 @@ parsed_source.tokens();
 // returns the module (AST)
 parsed_source.module();
 // returns the `SourceTextInfo`
-parsed_source.source();
+parsed_source.text_info();
 ```

